### ISSUE TITLE
[1.11.2] Error creating world with ATG

### ISSUE THREAD

**Issue by Eufranio** [SpongePowered/SpongeForge#1524]
When creating a world with ATG (Alternate Terrain Generator), this error will be printed to console:
https://hastebin.com/niqezigefo.vbs
```
>  [01:14:14] [Server thread/INFO]: [STDERR]: java.lang.reflect.InvocationTargetException
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at java.lang.reflect.Method.invoke(Method.java:498)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at org.spongepowered.common.registry.util.RegistryModuleLoader.tryAdditionalRegistration(RegistryModuleLoader.java:192)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at org.spongepowered.common.registry.SpongeGameRegistry.registerAdditionalPhase(SpongeGameRegistry.java:503)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at org.spongepowered.common.registry.SpongeGameRegistry.registerModulePhase(SpongeGameRegistry.java:497)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at org.spongepowered.common.registry.SpongeGameRegistry.init(SpongeGameRegistry.java:524)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at org.spongepowered.mod.SpongeMod.onInitialization(SpongeMod.java:256)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at java.lang.reflect.Method.invoke(Method.java:498)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.EventBus.post(EventBus.java:275)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:252)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:230)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at java.lang.reflect.Method.invoke(Method.java:498)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at com.google.common.eventbus.EventBus.post(EventBus.java:275)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraftforge.fml.common.LoadController.redirect$onPost$zzd000(LoadController.java:553)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:147)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:828)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:109)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:333)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:215)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:442)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at java.lang.Thread.run(Thread.java:748)
>  [01:14:14] [Server thread/INFO]: [STDERR]: Caused by: java.lang.NullPointerException
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1011)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at java.util.concurrent.ConcurrentHashMap.put(ConcurrentHashMap.java:1006)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     at org.spongepowered.common.registry.type.world.gen.BiomeTypeRegistryModule.registerAdditional(BiomeTypeRegistryModule.java:75)
>  [01:14:14] [Server thread/INFO]: [STDERR]:     ... 37 more
```
    Minecraft: 1.11.2
    SpongeAPI: 6.1.0-SNAPSHOT-806a22a
    SpongeForge: 1.11.2-2282-6.1.0-BETA-2343
    Minecraft Forge: 13.20.0.2298
Not running any plugins. Log: https://gist.github.com/Eufranio/ba4d9756851398f54b03ce269c4da3ad

**Comment by Eufranio** [comment]
This also happens starting the server with Biomes O' Plenty.

**Comment by maxanier** [comment]
Same for me with Vampirism (which adds one biome).

**Comment by Deamon5550** [comment]
I believe this was fixed by https://github.com/SpongePowered/SpongeCommon/commit/7178a358498a585f793abc431772d32c3be40bb1, although that might need backporting?

**Comment by phit** [comment]
Closing this issue, as of today 1.10.2 and 1.11.2 are not longer supported. If a sufficiently, decent, and tested Pull Request is made to fix this bug/issue, it will still be considered, however, it is NOT to be expected that any bug fixes made for 1.12.2 will be back ported to 1.11.2 or 1.10.2.

If you can reproduce this issue with SpongeForge on Minecraft 1.12.2 and this issue was wrongly closed, please [open a new issue](https://github.com/SpongePowered/SpongeForge/issues/new) with updated information.
